run-spread

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# check dependencies
+if ! snap list snapcraft &>/dev/null; then
+    echo "snapcraft is not installed"
+    exit 1
+fi
+
+# Clean the snaps created in previous runs
+touch test-build
+rm -rf built-snap
+mkdir built-snap
+rm -f snapd_1337.*.snap snapd_1337.*.snap.keep
+
+# Build snapd snap
+snapcraft --use-lxd
+for snap in built-snap/snapd_1337.*.snap; do
+    mv "${snap}" "${snap}.keep"
+done
```

Comment:
I just tested this manually, and there are no snaps in `build-snap`. The only snap which gets created is `snapd_1337.2.63+git450.g30be453_amd64.snap` in the project root, not in `built-snap`.

So in my testing nothing is found in this for loop and we don't end up with any files in `built-snap`.

Here's what I saw:

```
me@hostname:/tmp/snapd$ touch test-build
me@hostname:/tmp/snapd$ rm -rf built-snap
me@hostname:/tmp/snapd$ mkdir built-snap
me@hostname:/tmp/snapd$ rm -f snapd_1337.*.snap snapd_1337.*.snap.keep
me@hostname:/tmp/snapd$ snapcraft --use-lxd
Installed package repositories                                                                                                     
Generated snap metadata                                                                                                            
Created snap package snapd_1337.2.63+git450.g30be453_amd64.snap
me@hostname:/tmp/snapd$ find -name snapd_1337.*  
./snapd_1337.2.63+git450.g30be453_amd64.snap
me@hostname:/tmp/snapd$ ls -als built-snap
total 8
4 drwxrwxr-x  2 me me 4096 Jul 22 09:32 .
4 drwxrwxr-x 63 me me 4096 Jul 22 09:33 ..
```

Reply:
in prepare.sh in function build_snapd_snap we get the snap.keep from built-snap dir

Also you need to export TESTS_USE_PREBUILT_SNAPD_SNAP=true to get the snap